lib/fastlane/plugin/wpmreleasetoolkit/actions/common/extract_release_notes_for_version_action.rb

```diff
@@ -41,7 +41,7 @@ def self.extract_notes(release_notes_file_path, version)
 
       def self.check_and_commit_extracted_notes_file(file_path, version)
         Action.sh("git add #{file_path}")
-        Action.sh("git diff-index --quiet HEAD || git commit -m \"Update draft release notes for #{version}.\"")
+        Action.sh("git diff-index --quiet HEAD || git commit -m \"Update draft release notes for #{version}\"")
```

Comment:
Aside from the formatting, it would have been good to update the code to use either the Fastlane Git API or the git wrapper gem that I think we already have setup. 

But I'm in the middle of my support rotation and already feel guilty for indulging in this fix.

Reply:
Took the liberty to do exactly that in 72b0592d.

Tested by running `extract_release_notes_for_version` in a dummy branch of PCiOS, worked as expected (first run created the commit with changes to the release notes files, second call detected there was nothing new to commit)